dist/index.html

```diff
@@ -55,10 +55,15 @@ <h2>Search Results</h2>
           </select>
         </form>
       </div>
-        <section class="bookings-searched">
-          <!-- Search Results Populate Here -->
-        </section>
+      <section class="bookings-searched">
+        <!-- Search Results Populate Here -->
+      </section>
     </section>
+    <div class="background-overlay hidden">
+      <section class="confirmation">
+        
```

Comment:
Not a need to have, but nice to have: 
Include a comment here that confirmation will appear here

Reply:
Handled this and exit button in accessibility branch I started yesterday - great minds! 👍